pkg/kapp/diff/resource_with_history.go

```diff
@@ -214,3 +231,12 @@ func (resourceWithoutHistory) removeAppliedResAnnKeysMods() []ctlres.ResourceMod
 		},
 	}
 }
+
+func (resourceWithoutHistory) removeManagedFieldsResMods() []ctlres.ResourceMod {
+	return []ctlres.ResourceMod{
```

Comment:
good use of ResourceMod. i would recommend instead of having a dedicated method just keeping this as a variable within the method that applies that mod.

also since there is only one mod there is no need to do it in the loop above.